classroom_in_a_box/ks/\

```diff
@@ -0,0 +1,80 @@
+#platform=x86, AMD64, or Intel EM64T
```

Comment:
What is this file? Wasn't this only supposed to be a quote? (also, that's kind of a weird filename?)